### ISSUE TITLE
Poem is awesome!

### ISSUE THREAD

**Issue by danechitoaie** [poem-web/poem#674]
I just wanted to say that Poem framework is just really awesome. I think it's the best and most nicely organized framework out of all the Rust web frameworks (that I've tried). I'm still getting started so I still have many things to learn both about Poem and Rust, but Poem makes it all a nice experience overall. I think it also has the best support for OpenAPI.
So, just wanted to say thanks for it, and I hope it gets even more popular and more people will use it and it will continue to get supported and get new features and keep improving in the future as well.

**Comment by NexRX** [comment]
Couldn't agree more :D

**Comment by ollyde** [comment]
Yep, I tried many of them, Poem is still by far the best one because of the auto-docs.

**Comment by Navise** [comment]
Rust is going Crazy and this framework is 🔥
